pkg/apis/serving/k8s_validation_test.go

```diff
@@ -245,8 +245,7 @@ func TestContainerValidation(t *testing.T) {
 		c: corev1.Container{
 			Image: "foo",
 			Ports: []corev1.ContainerPort{{
-				Name:          "http1",
```

Comment:
I think you should keep at least one test with valid `ContainerPort`.

Reply:
Yes, good catch, ty